examples/swapi/pkg/api/people/get_by_id.go

```diff
@@ -0,0 +1,42 @@
+package people
+
+import (
+	"context"
+	"github.com/caicloud/nirvana/examples/swapi/pkg/model"
+	"sort"
+	"github.com/caicloud/nirvana/definition"
```

Comment:
import format shuold obey the go conventions.

Reply:
done